config.yml

```diff
@@ -0,0 +1,49 @@
+relaychain:
```

Comment:
Is the config.json still need?

Reply:
it can be used probably for testing as parachain-launch will take more time to launch and depends on docker